github3/orgs.py

```diff
@@ -508,6 +508,20 @@ def public_events(self, number=-1, etag=None):
         url = self._build_url('events', base_url=self._api)
         return self._iter(int(number), url, Event, etag=etag)
 
+    @requires_auth
+    def invitations(self, number=-1, etag=None):
+        r"""Iterate over outstanding invitations to this organization.
+
+        Implements https://developer.github.com/v3/orgs/members/#list-pending-organization-invitations
+
+        :returns: generator of :class:dict
+        """
+        headers = {'Accept': 'application/vnd.github.korra-preview', }
+        params = {}
+        url = self._build_url('invitations', base_url=self._api)
+        return self._iter(int(number), url, dict, params=params, etag=etag,
```

Comment:
I think noting that this is a preview feature and it may change would be sufficient to allow us to introduce an object.